### ISSUE TITLE
 rctr_state file density values of the gas phase

### ISSUE THREAD

**Issue by wittregr** [VlachosGroup/openmkm#24]
In the rctr_state file, are the reported density values the density of the gas phase? If not, please correct the documentation.

**Comment by MaxRCohen** [comment]
Hi @mbkumar, I see you reduced the clause of "density of the gas phase" to be just "density" in the documentation. Can you specify what it is the density of? Is it the gas phase, or the density of some other quantity?

**Comment by skasiraj** [comment]
Yes that is correct. The quantity being printed here is the density of reactor contents. Cantera documentation says "Returns the current density (kg/m^3) of the reactor's contents." I presume this is the gas-phase density from the ideal-gas law as we are using the Ideal-Gas assumption for the gas-phase species.

**Comment by MaxRCohen** [comment]
Great, this specification should be included in the documentation for clarity since just "density" can be ambiguous.
